platform/src/main/java/org/hillview/table/api/IDurationColumn.java

```diff
@@ -59,4 +62,12 @@ public int compare(final Integer i, final Integer j) {
             }
         };
     }
+
+    @Override
+    default long hashCode64(int rowIndex, long seed) {
+        if (isMissing(rowIndex))
+            throw new MissingException(this, rowIndex);
+        final Duration tmp = this.getDuration(rowIndex);
+        return LongHashFunction.xx(seed).hashLong(tmp.toMillis());
```

Comment:
You the suitable Converter method to convert this to a double, and then apply the same algorithm as for doubles.